log/logger.go

```diff
@@ -49,21 +50,27 @@ type Logger interface {
 }
 
 // logger provides default implementation for Logger. It logs using Go log API
+// mutex is needed in cases when multiple clients run concurrently
 type logger struct {
 	prefix   string
 	logLevel uint
+	sync.Mutex
```

Comment:
```suggestion
	lock sync.Mutex
```
`Logger` has no functionality related to Mutex, so embedding (inheriting) is not needed.